pkg/cloud/aws/actuators/scope.go

```diff
@@ -39,6 +39,8 @@ var (
 	sessionCache sync.Map
 )
 
+const apiEndpointPort = 443
```

Comment:
I believe this should probably be 6443: https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/1f9a31c0a17d3637a570e43171c9020c9a06379e/pkg/cloud/aws/services/elb/loadbalancer.go#L149

Reply:
On a tangential note, Instead of hardcoding the value here and in the load balancer creation we really should take into account the ClusterConfiguration: https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/master/pkg/apis/awsprovider/v1alpha1/awsclusterproviderconfig_types.go#L60

That is something we should treat as a followup item, though.

Reply:
Port change done